packages/main/src/TabContainer.hbs

```diff
@@ -3,7 +3,7 @@
 	dir="{{rtl}}"
 >
 	<div class="{{classes.header}}" id="{{_id}}-header">
-		<ui5-icon @click="{{_onHeaderBackArrowClick}}" class="{{classes.headerBackArrow}}" src="sap-icon://slim-arrow-left" tabindex="-1" accessible-name="Previous" icon-tooltip="true"></ui5-icon>
+		<ui5-icon @click="{{_onHeaderBackArrowClick}}" class="{{classes.headerBackArrow}}" src="sap-icon://slim-arrow-left" tabindex="-1" accessible-name="{{previousIconACCName}}" show-tooltip="true"></ui5-icon>
 
```

Comment:
`show-tooltip` should be enough as it is a Boolean property, replace `show-tooltip="true"`  with `show-tooltip` here and below

Reply:
Done